bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -13,14 +13,18 @@
 """ Easy-to-use wrapper for properly controlling Spot """
 import os
 import os.path as osp
+import pdb
 import time
 from collections import OrderedDict
 
 import bosdyn.client
 import bosdyn.client.lease
 import bosdyn.client.util
 import cv2
+import magnum as mn
 import numpy as np
+import quaternion
+import sophus as sp
```

Comment:
same as above

Reply:
Will test and add doc as a new PR after trying a fresh env setup.